src/Entity/OgMembership.php

```diff
@@ -561,4 +591,39 @@ public function isOwner(): bool {
     return $group instanceof EntityOwnerInterface && $group->getOwnerId() == $this->getOwnerId();
   }
 
+  /**
+   * Returns the value of a given untranslatable single property field.
+   *
+   * This method is optimized for dealing with untranslatable fields with a
+   * single value stored in a single property. For performance reasons this
+   * avoids calling ::get() so we can skip the expensive construction of the
+   * FieldItemList and the slow operations that are done to determine
+   * translatability and cardinality of the field, such as loading and
+   * inspecting the field and field storage definitions.
+   *
+   * @param string $field
+   *   The name of the field for which to return the value.
+   * @param string $property
+   *   The name of the field property that holds the value. Defaults to 'value'.
+   *
+   * @return mixed|null
+   *   The value, or NULL if no value has been set.
+   */
+  protected function getUntranslatedSinglePropertyFieldValue(string $field, string $property = 'value') {
```

Comment:
Maybe try the implementation I have in the core issue, with a todo to that issue?

Reply:
I added a remaining task to the core issue so we don't forget this.